steward/tests/acceptance.rs

```diff
@@ -30,65 +35,436 @@ use steward::config::StewardConfig;
 /// invocations as `cargo test` executes tests in parallel by default.
 pub static RUNNER: Lazy<CmdRunner> = Lazy::new(|| CmdRunner::default());
 
-/// Use `StewardConfig::default()` value if no config or args
+/// Use command-line argument value
 #[test]
-#[ignore]
-fn start_no_args() {
+fn eth_keys_add() -> io::Result<()> {
+    let key_dir = TempDir::new("test_key")?;
```

Comment:
For these tests we don't need two tempdirs, we can make one and then base all of our file paths off of that as the root.

Reply:
This commit should fix this: e71050632279eadbabf080f2df7d9ff69bc71f5c